grpc-server/hardware.go

```diff
@@ -241,15 +240,16 @@ func (s *server) DeprecatedWatch(in *hardware.GetRequest, stream hardware.Hardwa
 	}
 }
 
-// Cert returns the public cert that can be served to clients
-func (s *server) Cert() []byte {
-	return s.cert
-}
+// TODO: is this used anywhere?
+// // Cert returns the public cert that can be served to clients
+// func (s *server) Cert() []byte {
+// 	return s.cert
+// }
 
-// ModTime returns the modified-time of the grpc cert
-func (s *server) ModTime() time.Time {
-	return s.modT
-}
+// // ModTime returns the modified-time of the grpc cert
+// func (s *server) ModTime() time.Time {
+// 	return s.modT
+// }
```

Comment:
As far as I could tell from a quick glance, these didn't actually appear to be used anywhere.